Orange belt refactoring (#7)
AppSecure-nrw/security-belts: orange/team-has-a-security-champion.md

# Team has a Security Champion

Find and train a [Security Champion](https://github.com/AppSecure-nrw/security-belts#security-champion) from within the team.

> *Note*: The Security Champion should be a volunteer interested in software security. Just forcing someone to take the role is worse than having none.

Reviewer: ```suggestion
*Note*: The Security Champion should be a volunteer interested in software security. Just forcing someone to take the role is worse than having no Security Champion.
```

## Benefits

> - The Security Champion brings security knowledge to the team so that the team can take responsibility for security.

Reviewer: ```suggestion
- The Security Champion brings security knowledge to the team so that the team can make informed security decisions and take responsibility for security.
```

- Central security experts can focus on other tasks.

## Assessment

> - Show that the Security Champion has already completed a security training.

Reviewer 2: ```suggestion
- Show that the Security Champion has completed a security training.
```

> - Show the time invested by the Security Champion.

Reviewer 2: ```suggestion
- Show the improvements and activities made by the Security Champion, e.g., training the team and executing belt activities.
```

> ## Further Reading

Reviewer 2: ```suggestion
## Exemplary Vendors
```

> - [Fraunhofer IEM - Security Champion Training](https://www.iem.fraunhofer.de/de/academy/schulungsangebot/security-champion-training.html)

Reviewer 2: ```suggestion
- Finding Security Champions in Blends of Organisational Culture by Ingolf Becker et al, 2017
- Selecting Security Champions by Trevor Gabriel et al., 2011

## Exemplary Vendors
- [Fraunhofer IEM - Security Champion Training](https://www.iem.fraunhofer.de/de/academy/schulungsangebot/security-champion-training.html)
```

# Team has a Security Champion

Find and train a [Security Champion](https://github.com/AppSecure-nrw/security-belts#security-champion) from within the team.

*Note*: The Security Champion should be a volunteer interested in software security. Just forcing someone to take the role is worse than having no Security Champion.

## Benefits

- The Security Champion brings security knowledge to the team so that the team can make informed security decisions and take responsibility for security.
- Central security experts can focus on other tasks.

## Assessment

- Show that the Security Champion has completed a security training.
- Show the improvements and activities made by the Security Champion, e.g., training the team and executing belt activities.

> ## Further Reading
> - Finding Security Champions in Blends of Organisational Culture by Ingolf Becker et al, 2017
> - Selecting Security Champions by Trevor Gabriel et al., 2011
> 
> ## Exemplary Vendors
> - [Fraunhofer IEM - Security Champion Training](https://www.iem.fraunhofer.de/de/academy/schulungsangebot/security-champion-training.html)

Reviewer 2: ```suggestion
## Further Readings
- Literature: Finding Security Champions in Blends of Organisational Culture by Ingolf Becker et al, 2017
- Literature: Selecting Security Champions by Trevor Gabriel et al., 2011
- [Training: Fraunhofer IEM - Security Champion Training](https://www.iem.fraunhofer.de/de/academy/schulungsangebot/security-champion-training.html)
```
